bin/tldr

```diff
@@ -24,7 +25,8 @@ program
   .option('-e, --random-example', 'Show a random example')
   .option('-f, --render [file]', 'Render a specific markdown [file]')
   .option('-m, --markdown', 'Output in markdown format')
-  .option('-p, --platform [type]', `Override the current platform [${platforms.supportedPlatforms.join(', ')}]`);
+  .option('-p, --platform [type]', `Override the current platform [${platforms.supportedPlatforms.join(', ')}]`)
+  .option('completion <shell>', 'Generate and add shell completion script to your shell configuration');
```

Comment:
```suggestion
  .option('-c, --completion [shell]', 'Generate and add shell completion script to your shell configuration');
```
Wouldn't it better to use the same syntax as for other options?

Reply:
I believe the latter option looks better. I didn't add `--` to the completion command because, although it's not technically correct, I wanted to express it as a subcommand.

I initially used `<shell>` to indicate a required argument, but I realized it becomes meaningless when using `.option()`. It seems better to align with the convention by using `[shell]`.

In the case of the former option, the short `-c` is similar to `.option('-c, --clear-cache', 'Clear the local cache');`, so we might need to reconsider this.

Reply:
For me the second option is fine

Reply:
Changed to `[shell]` for consistency.